tools/flowc/flowc_usage.flow

```diff
@@ -54,6 +54,11 @@ printUsage(config) {
 	prih(help,"      java-main=<name>          Specify the java main class for the project. By default it is a name of a programm");
 	prih(help,"      java-incremental=1        Switch on incremental java target generation");
 	prih(help,"      java-library=id1,id2,...  Build a java library with id1, id2, ... interface");
+	prih(help,"      java-builder=<builder>    Use a specified build system. Now available: gradle or none");
+	prih(help,"      java-incremental=1        Do not re-generate modules, which were not changed.");
+	prih(help,"      java-dependencies=<dep1,dep2,...>  External java libraries, used by java natives.");
```

Comment:
Is this for all build systems?

Reply:
No, currently only for java. Potentially we could use npm for JS to fetch JS dependencies, but it seems to be quite different. Also there's a conan package manager for c++, but I'm not sure that it's needed now.